src/librustc_codegen_ssa/Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [package]
+
```

Comment:
Accidential?

Reply:
I fixed this when I pulled Alex's patch into my repo.